orion/pkg/repos/migrations/000007_create_table_users.up.sql

```diff
@@ -9,8 +9,8 @@ CREATE TABLE users
     middle_name varchar(32),
     email       varchar(64)  NOT NULL,
     phone       varchar(24)  NOT NULL,
-    isGuardian  boolean      NOT NULL DEFAULT 0,
-    guardianId  int unsigned,
+    is_guardian boolean      NOT NULL DEFAULT 0,
+    guardian_id int unsigned,
```

Comment:
if you end up changing the migration file, your current sql might have already created the user table with `isGuardian` and `guardianId` columns. To fix this, you will have to fix your migration state. Use these commands:
```
migrate -source file://pkg/repos/migrations -database mysql://USER:PASSWORD@\(localhost\)/mathnavdb down 1
migrate -source file://pkg/repos/migrations -database mysql://USER:PASSWORD@\(localhost\)/mathnavdb up 1
```
This will move your migration from version 7 to 6 (drops the user table). Then it will move you back to 7 (create user table again).